flink-runtime-web/web-dashboard/src/app/interfaces/job-checkpoint.ts

```diff
@@ -164,6 +164,7 @@ export interface CheckPointSubTaskInterface {
       buffered: CheckPointMinMaxAvgStatisticsInterface;
       duration: CheckPointMinMaxAvgStatisticsInterface;
     };
+    start_delay: CheckPointMinMaxAvgStatisticsInterface;
```

Comment:
Thanks for pointing this out @vthinkxie. I've added a single line entry in the `src/app/interfaces/job-checkpoint.ts`. Is that what you had in mind?

Reply:
lgtm